### PR TITLE
Add master checkbox to toggle all import selections

### PR DESCRIPTION
**Pull request by FagnerMartinsBrack**
## Summary
This PR adds a "select all" master checkbox to the import review page that allows users to quickly select or deselect all imported URLs at once. The checkbox displays three states: fully selected, fully deselected, and partially selected (indeterminate).

## Key Changes

- **New Route**: Added `POST /import/:id/toggle-all` endpoint to handle bulk selection/deselection of all import rows
- **Store Implementation**: Implemented `toggleAllImportSelection` method in both in-memory and DynamoDB import session stores
- **UI Components**: 
  - Added master checkbox form in the import template with proper state indicators
  - Created client-side module (`import.client.ts`) to handle the HTML5 `indeterminate` property, which cannot be set via attributes
  - Updated view model to track selection states (`allSelected`, `noneSelected`, `someSelected`)
- **Styling**: Added CSS classes for the select-all form and label layout
- **Validation**: Added `ImportToggleAllSchema` for request body validation
- **Client Bundle**: Configured esbuild to bundle and auto-initialize the indeterminate checkbox handler

## Implementation Details

- The master checkbox state is determined by comparing total selected items against total URLs
- The indeterminate state is applied via JavaScript since HTML attributes cannot represent this state
- The client-side handler re-applies indeterminate state after htmx swaps to maintain the property on newly-inserted DOM nodes
- Bulk toggle operations preserve the current page number in redirects
- All operations include proper authorization checks (userId validation)
- Comprehensive test coverage for all three checkbox states and edge cases (pagination, malformed requests, etc.)

https://claude.ai/code/session_01LtAopXzBNVgMrZj1tPhQpB

### CODE REVIEW

**Comment by FagnerMartinsBrack** [comment]
@claude Review this PR. Use your Read tool to read the local file `.github/workflows/claude-PR-code-reviewer.md` for instructions. The file is in the checked-out repository.

## Context
- **Repo**: Readplace/readplace.com
- **PR**: #241
- **Commit**: 0fa717a8a7bb36ae0a707716e36ab5458d971165

**Comment by FagnerMartinsBrack** [comment]
@claude address all low prio issues

**Comment by FagnerMartinsBrack** [comment]
@claude Review this PR. Use your Read tool to read the local file `.github/workflows/claude-PR-code-reviewer.md` for instructions. The file is in the checked-out repository.

## Context
- **Repo**: Readplace/readplace.com
- **PR**: #241
- **Commit**: aec65fc4b734b7620ebb4e7b3cd4083262b8ceeb

File: projects/hutch/src/runtime/providers/import-session/dynamodb-import-session.ts [hunk 1]
```diff
@@ -107,16 +112,35 @@ export function initDynamoDbImportSession(deps: {
...
+				},
+			});
+		},
+		toggleAllImportSelection: async ({ id, userId, checked }) => {
+			const row = await loadOwned(id, userId);
+			if (!row) return;
+			await table.update({
+				Key: { sessionId: id },
+				ConditionExpression: "userId = :uid",
+				UpdateExpression: "SET deselected = :d, allDeselected = :a",
```
Comment (FagnerMartinsBrack) [inline projects/hutch/src/runtime/providers/import-session/dynamodb-import-session.ts]: @claude to avoid double boolen (not !not) which makes my head spin, use a positive allSelected in the DB and add negative conditions in code (or whatever works to reduce the flag cyclomatic complexity)

**Comment by FagnerMartinsBrack** [comment]
@claude Resolve conflicts. Use your Read tool to read the local file `.github/workflows/claude-PR-conflict-fixer.md` for instructions. The file is in the checked-out repository.

## Context
- **Repo**: Readplace/readplace.com
- **PR**: #241
- **Branch**: claude/select-all-checkbox-feature-bSAGM

**Comment by FagnerMartinsBrack** [comment]
@claude Review this PR. Use your Read tool to read the local file `.github/workflows/claude-PR-code-reviewer.md` for instructions. The file is in the checked-out repository.

## Context
- **Repo**: Readplace/readplace.com
- **PR**: #241
- **Commit**: 16ddda75394c0cdc6afd138f077f3dff2649ed2e

**Comment by FagnerMartinsBrack** [comment]
@claude address low prio issue

**Comment by FagnerMartinsBrack** [comment]
@claude Review this PR. Use your Read tool to read the local file `.github/workflows/claude-PR-code-reviewer.md` for instructions. The file is in the checked-out repository.

## Context
- **Repo**: Readplace/readplace.com
- **PR**: #241
- **Commit**: fc9397e1f9cfffe1e2a186ef70d79cdac11850ce
